src/mon/ConfigMap.cc

```diff
@@ -250,6 +254,58 @@ void ConfigMap::parse_key(
   }
 }
 
+int ConfigMap::add_option(
+  CephContext *cct,
+  const std::string& name,
```

Comment:
why not std::string_view?

Reply:
I agree that we should use std::string_view here. However, I think that this change is out of the scope of this PR as it will require additional changes to already existing users that depend on std::string. (e.g `string value = orig_value;`)